### DISCUSSION TITLE
Duda entrega MacoWins

### DISCUSSION THREAD

**Discussion by Schmidt-mat** [dds-jv/foro#341] [Consultas generales] [answered]
Buenas, tengo entendido que después de haber entregado por primera vez el ejercicio de MacoWins hay que entregarlo otra vez en su versión "final", pero el link de "TPI - fecha límite: 24/4" que es este: https://classroom.github.com/classrooms/4492795-dds-2022-jueves no se encuentra disponible.

**Comment by gastonprieto** [comment, marked as answer]
buenas @Schmidt-mat 

listo, fixee el link, muchas gracias por el aviso
